doc/modules/lmnn.rst

```diff
@@ -80,72 +52,59 @@ in the constructor.
 Mathematical formulation
 ========================
 
-The LMNN objective function consists of two competing terms, the pull loss
-that pulls target neighbors closer to their reference samples and the push
-loss that pushes impostors away:
+LMNN learns a linear transformation matrix :math:`L` of size ``(n_features, n_features_out)``.
```

Comment:
I think the shape of `L` should be ``(n_features_out, n_features)`` as it is in both the theoretical objective definition and the implementation.